packages/daisyui/src/styles.css

```
@@ -0,0 +1,9 @@
+@import "tailwindcss/preflight";
+@import "tailwindcss/utilities";
+
+@layer base {
+  :root {
+    --color-primary: 255 115 179;
+    --color-secondary: 111 114 185;
+  }
+} 
```

Adding blank line at end to remove the red circled dash
```suggestion
} 

```